vn.hoabinh.quan.CoBang.appdata.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This should be part of the upstream source, it's not specific to Flatpak

Reply:
Ok. Will remove this file.

It is in upstream, but because when I see how other apps submit to FlatHub, I saw they include AppData, I though it is needed.